src/tests/nested-containers.test.ts

```diff
@@ -0,0 +1,650 @@
+
+        expect(transaction.rows[0].mst).toEqual([{'key': 1, 'value': [10, 12, 16]}, {'key': 2, 'value': [200, 300]}]);
+
+        expect(transaction.rows[0].mv).toEqual([{'key': 1, 'value': [10, 10, 12, 16]}, {'key': 2, 'value': [200, 300]}]);
+
+        expect(transaction.rows[0].mo).toEqual([{'key': 10, 'value': 1000}, {'key': 11, 'value': null}]);
+
+        expect(transaction.rows[0].mm).toEqual([{'key': 10, 'value': [{'key': 200, 'value': 2000}, {'key': 201, 'value': 2001}]}, {'key': 11, 'value': [{'key': 300, 'value': 3000}, {'key': 301, 'value': 3001}]}]);
+
+        expect(transaction.rows[0].mp).toEqual([{'key': 36, 'value': {'key': 300, 'value': 301}}, {'key': 37, 'value': {'key': 600, 'value': 601}}]);
+
+        expect(transaction.rows[0].mt).toEqual([{'key': 1, 'value': {'field_0': 10, 'field_1': 11}}, {'key': 2, 'value': {'field_0': 200, 'field_1': 300}}]);
+
+        expect(transaction.rows[0].pst).toEqual({'key': 20, 'value': [200, 202]});
+
+        expect(transaction.rows[0].pv).toEqual({'key': 10, 'value': [100, 100, 102]});
+
+        expect(transaction.rows[0].po).toEqual({'key': 70, 'value': 71});
+        expect(transaction.rows[1].po).toEqual({'key': 70, 'value': null});
+
+        expect(transaction.rows[0].pm).toEqual({'key': 6, 'value': [{'key': 20, 'value': 300}, {'key': 21, 'value': 301}]});
+
+        expect(transaction.rows[0].pp).toEqual({'key': 30, 'value': {'key': 301, 'value': 302}});
+
+        expect(transaction.rows[0].pt).toEqual({'key': 10, 'value': {'field_0': 100, 'field_1': 101}});
+
+        expect(transaction.rows[0].tst).toEqual({'field_0': 10, 'field_1': [21, 31], 'field_2': [41, 51, 61]});
+
+        expect(transaction.rows[0].tv).toEqual({'field_0': 16, 'field_1': [26, 36], 'field_2': [46, 506, 606]});
+
+        expect(transaction.rows[0].to).toEqual({'field_0': 100, 'field_1': null, 'field_2': 200, 'field_3': null, 'field_4': 300});
+        expect(transaction.rows[1].to).toEqual({'field_0': null, 'field_1': null, 'field_2': 10, 'field_3': null, 'field_4': 20});
+
+        expect(transaction.rows[0].tm).toEqual({'field_0': 126, 'field_1': [{'key': 10, 'value': 100}, {'key': 11, 'value': 101}], 'field_2': [{'key': 80, 'value': 800}, {'key': 81, 'value': 9009}]});
+
+        expect(transaction.rows[0].tp).toEqual({'field_0': 127, 'field_1': {'key': 18, 'value': 28}, 'field_2': {'key': 19, 'value': 29}});
+
+        expect(transaction.rows[0].tt).toEqual({'field_0': {'field_0': 1, 'field_1': 2}, 'field_1': {'field_0': 30, 'field_1': 40}, 'field_2': {'field_0': 50, 'field_1': 60}});
+
+        expect(transaction.rows[0].vos).toEqual([{'_count': 18, '_strID': 'dumstr'}, null, {'_count': 19, '_strID': 'dumstr'}]);
+
+        expect(transaction.rows[0].pvo).toEqual({'first': 183, 'second': [100, null, 200]});
+    });
+
+    it('should test Key-Value nested containers', async () => {
+        jest.setTimeout(30000);
+
+        await api.getAbi('nestcontn2kv');
+
```

Comment:
same as previous api.getApi(), here please add similar comment to explain nestcotn2kv and where is the related nested_container_kv.cpp contract